rmf_schedule_visualizer/src/VisualizerData.hpp

```diff
@@ -103,6 +108,10 @@ class VisualizerDataNode : public rclcpp::Node
   std::unordered_map<
     rmf_traffic::schedule::Version,
     std::vector<rmf_traffic::schedule::ParticipantId>> _conflicts;
+
+  rmf_traffic_ros2::schedule::WriterPtr _writer;
+  rmf_utils::optional<rmf_traffic_ros2::schedule::MirrorManager> _mirror_mgr;
```

Comment:
Is there a reason for storing a separate mirror manager for negotiations? If not would it be possible to use `data->mirror`?

Reply:
nice catch, removed ed947f0